### ISSUE TITLE
Update/cleanup translations and move to MT 5.0.0+ translation system

### ISSUE THREAD

**Issue by OgelGames** [mt-mods/technic#104]
It needs to be done at some stage...

**Comment by BuckarooBanzay** [comment]
I don't have much experience with the translation system in general but it looks like the mod it is prepared for that:
The `locale` directory with the properly formatted files is there and the `technic.getter` does pretty much the same thing as `minetest.get_translator()` does

**Comment by OgelGames** [comment]
Yeah, it's almost the same, the updating/cleanup will be harder than changing the translator.

Here is an example (which is what reminded me about it): https://github.com/ChaosWormz/teleport-request/pull/39/files

**Comment by BuckarooBanzay** [comment]
Some work may have been done here: https://github.com/minetest-mods/technic/pull/529

**Comment by OgelGames** [comment]
That will help a lot 👍 Of course there will still be some work to do because of all the changes we have made...

Also probably better to wait until the network and chests updates are done before this.

**Comment by S-S-X** [comment]
I think it would be better to close this issue and create multiple more specific issues for translations.

Bit more specific issue descriptions and narrowing individual issue scope can make workflow very beginner friendly / not require deep technical knowledge about underlying translation system.

Also increases visibility for progress tracking purposes.
